### PR TITLE
fix(frontend): Render run parameters in run details tab.

### PR DESCRIPTION
**Pull request by jlyaoyuli**
Before:
Screenshot 2023-04-24 at 10 38 21 AM

After:
Screenshot 2023-04-24 at 10 38 58 AM

### CODE REVIEW

File: frontend/src/pages/RunDetailsV2.test.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ describe('RunDetailsV2', () => {
       pipeline_id: 'some-pipeline-id',
       pipeline_manifest: '{some-template-string}',
     },
-    runtime_config: { parameters: [{ name: 'param1', value: 'value1' }] },
+    runtime_config: { parameters: { param1: 'value1' } },
```
[the remark this answers is not in the document]
Comment (jlyaoyuli) [reply frontend/src/pages/RunDetailsV2.test.tsx:91]: I remember the latest proto is using `parameters`.
https://github.com/kubeflow/pipelines/blob/master/backend/api/v2beta1/runtime_config.proto#L23-L33

[the remark this answers is not in the document]
Comment (chensun) [reply frontend/src/pages/RunDetailsV2.test.tsx:91]: I see, so this maps API proto but not IR proto. It's very confusing that we have mismatched names in IR and API...

**Review by chensun — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**